### DISCUSSION TITLE
New emfjson release planned?

### DISCUSSION THREAD

**Discussion by mdlnr** [eclipse-emfcloud/emfcloud#166] [EMFJson Jackson] [answered]
Hi,

since the last release in December some relevant changes have been committed in the emfjson project. Can you estimate when the next release will happen?

Best regards
Thorsten

**Comment by martin-fleck-at** [comment, marked as answer]
Hi Thorsten,

We are looking into a few more issues but as soon as that is resoled, we can create a new release. So I believe we should be able to do that by mid August. Until then you can always use the latest snapshot versions:
- MVN: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/emfcloud/emfjson-jackson/2.1.0-SNAPSHOT/
- P2: https://download.eclipse.org/emfcloud/emfjson-jackson/p2/nightly/

**Reply by mdlnr** [reply]
Thanks - mid August sounds fine.
